src/components/theme/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -1,20 +1,14 @@
-/**
- * Breadcrumbs components.
- * @module components/theme/Breadcrumbs/Breadcrumbs
- */
-
-import React, { Component } from 'react';
+import { useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import { compose } from 'redux';
 import { Link } from 'react-router-dom';
 import { Breadcrumb, Container, Segment } from 'semantic-ui-react';
-import { defineMessages, injectIntl } from 'react-intl';
+import { defineMessages, useIntl } from 'react-intl';
+import { useDispatch } from 'react-redux';
 
-import { Icon } from '@plone/volto/components';
 import { getBreadcrumbs } from '@plone/volto/actions';
 import { getBaseUrl, hasApiExpander } from '@plone/volto/helpers';
-
+import { useBreadcrumbs } from '@plone/volto/hooks/breadcrumbs/useBreadcrumbs';
```

Comment:
You can export this in https://github.com/plone/volto/blob/master/src/hooks/index.js 
and then directly import from `@plone/volto/hooks`